cli.js

```diff
@@ -13,6 +13,7 @@ const cli = meow(`
 
 	Options
 	  --fix             Automagically fix issues
+	  --fix-dry-run     Automatically fix problems without saving the changes to the file system
```

Comment:
I feel like it's unclear to the user how this would even work. The user will ask themselves: "If it does not save to the file system, what's the point of it?"

Reply:
--fix-dry-run     Automatically fix problems without saving the changes to the file system.
                      Because the default formatter does not output the fixed code, you'll have to use another one (e.g. json)
                      This flag can be useful for integrations which need to autofix text from command line without saving it to the filesystem.

How about the above one?
I'm worried about only this option's description looks too verbose because the other options are all single lines.
These descriptions are copy-pasted from https://eslint.org/docs/user-guide/command-line-interface#--fix-dry-run

Reply:
Perhaps "Preview fixed file contents" or "Print fixed file contents"?

```suggestion
	  --fix-dry-run     Preview fixed file contents
```